templates/items/race.hbs

```diff
@@ -0,0 +1,67 @@
+<form class="{{cssClass}} flexcol" autocomplete="off">
```

Comment:
Should perhaps consider naming the file `race-sheet.hbs` or `race-config.hbs` instead of just "race".